receive.go

```diff
@@ -26,15 +26,18 @@ type ReceiveOpt struct {
 	Merge         bool
 	Filter        FilterFunc
 	Differ        DiffType
+
+	diskWriterOpt DiskWriterOpt
```

Comment:
added explicit configuration attribute to work around the necessity to run the test as root (see below).